<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>753</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Conway's Game of Life</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>logo.bmp</normaloff>logo.bmp</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="GraphicsLayoutWidget" name="PlotView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>350</width>
      <height>350</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="botonStartPause">
    <property name="geometry">
     <rect>
      <x>63</x>
      <y>384</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Start / Pause</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botonRestart">
    <property name="geometry">
     <rect>
      <x>324</x>
      <y>384</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Reload</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tick">
    <property name="geometry">
     <rect>
      <x>23</x>
      <y>384</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Tick</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botonOpen">
    <property name="geometry">
     <rect>
      <x>242</x>
      <y>384</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>add .cells</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botonClean">
    <property name="geometry">
     <rect>
      <x>176</x>
      <y>384</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Clean</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="infoBox">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>250</y>
      <width>341</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="sizeOptions">
    <property name="geometry">
     <rect>
      <x>164</x>
      <y>412</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimum">
     <number>5</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>20</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>163</x>
      <y>435</y>
      <width>60</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Size NxN</string>
    </property>
   </widget>
   <widget class="GraphicsLayoutWidget" name="PlotView2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>20</y>
      <width>341</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="speedSlide">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>417</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1800</number>
    </property>
    <property name="value">
     <number>1000</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>65</x>
      <y>433</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Speed</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="xPosValue">
    <property name="geometry">
     <rect>
      <x>298</x>
      <y>414</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="yPosValue">
    <property name="geometry">
     <rect>
      <x>347</x>
      <y>414</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>282</x>
      <y>416</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>X : </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>331</x>
      <y>416</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Y :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>231</x>
      <y>414</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Load at</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>144</x>
      <y>416</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>404</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>_</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>380</y>
      <width>341</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Info</string>
    </property>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>321</width>
       <height>61</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Tarea 06 &lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Python Científico&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Diego Carrasco Vásquez&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open .cell</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionExit_2">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionRetina">
   <property name="text">
    <string>Retina</string>
   </property>
  </action>
  <action name="actionKortx">
   <property name="text">
    <string>Kortx</string>
   </property>
  </action>
  <action name="actionKortX">
   <property name="text">
    <string>KortX</string>
   </property>
  </action>
  <action name="actionConway_s_Game_of_Life">
   <property name="text">
    <string>Conway's Game of Life</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphicsLayoutWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
